core/src/test/java/feign/LoggerTest.java

```diff
@@ -72,18 +74,18 @@ public LogLevelEmitsTest(Level logLevel, List<String> expectedMessages) {
       return Arrays.asList(new Object[][] {
           {Level.NONE, Collections.emptyList()},
           {Level.BASIC, Arrays.asList(
-              "\\[SendsStuff#login\\] ---> POST http://localhost:[0-9]+/ HTTP/1.1",
+              "\\[SendsStuff#login\\] ---> POST http://localhost:[0-9]+/ HTTP",
               "\\[SendsStuff#login\\] <--- HTTP/1.1 200 OK \\([0-9]+ms\\)")},
           {Level.HEADERS, Arrays.asList(
-              "\\[SendsStuff#login\\] ---> POST http://localhost:[0-9]+/ HTTP/1.1",
```

Comment:
can we set the protocol so it matches the original log?

Reply:
done

I have restored old log messages for request line.